### DISCUSSION TITLE
音樂透露你的個性！想了解一個人，先看他的歌單

### DISCUSSION THREAD

**Discussion by lyien1021** [1101-GS4719-C/Python-Community-GS4719-C#1303] [General]
https://techteller.com/sci/music-psychology-and-personality/

身為一個熱愛音樂的人，我一天不聽音樂便渾身難受，而最近我突然想到，每個人都有自己偏好的音樂類型，而這跟每個人的性格會不會有一些關聯性？於是我上網查資料，發現此文章提到大腦與音樂之間的關係，以及各種風格的音樂分別可能代表的性格特質，還有一個很有趣的「音樂治療」，大家有興趣也可以看看。

最後順便分享我自己的音樂分享帳號：
https://www.instagram.com/invites/contact/?i=17a0564bf6etz&utm_content=ml60c1o
雖然最近因為太忙所以很少更新，不過如果有興趣的話還是可以看看😆或是跟我分享你喜歡的歌之類的

**Comment by YoYooHuang** [comment]
我發現多聽音樂真的有助於心情的舒緩！不同性質的歌曲，都會起到不同的作用。像我喜歡聽輕快的歌曲，這可以讓我的心情更加愉悅，做起事來更加有動力！

**Comment by orianna901001** [comment]
我常常會有想把自己的歌單改成全都是英文的，但久而久之，歌單裡還是都是日文歌，讀書時一整個讓人心情比較愉悅，但聽到英文歌時，真的很挑歌，對我來說英文歌比較適合開車出遊的時候~

**Comment by zilinng** [comment]
我比較喜歡慢一點的抒情歌，因為我聽歌的時候通常是想要自己一個人安靜待著休息，慢歌可以讓我的心情比較放鬆平靜，一直以來都很喜歡聽陳綺貞的我喜歡上你時的內心活動，可以去聽聽看～

**Comment by llh26** [comment]
當我心情比較低落的時候，反而喜歡聽一些比較sad的音樂，如果要我聽什麼嗨歌，我可能會更煩躁⋯，心情好才會聽歡快一點的音樂。我發現有些人聽歌好像不太會去注意歌詞，比較注重旋律是否合胃口，但我非常推薦大家去看歌詞，會發現更多細節，聽歌會更有感覺😽

**Comment by slps101023** [comment]
音樂和性格之間的確有一些有趣的聯繫！研究顯示，喜愛不同音樂類型的人可能有相似的性格特質。例如，喜歡古典音樂的人通常偏向沉穩和內向，而喜歡搖滾音樂的人則可能更具創造力和自信。至於音樂治療，這是一門利用音樂來促進情緒健康、減輕壓力和提升專注力的學問，已被應用於各種情境中，包括焦慮、抑鬱和創傷治療等。音樂的節奏、旋律和和聲都能影響我們的情緒和心理狀態，甚至可能幫助我們更了解自己。

**Comment by ShuliANN** [comment]
音樂的確很能反映我們的情緒狀態和人的個性也有相關性。很多時候心情好時可能會選擇輕快的音樂，而心情低落時則可能更偏向輕柔或治療性的歌曲。根據心情調整歌單，能讓音樂更好地幫助我們表達和處理情緒，這也是音樂的一大魅力所在！

**Comment by 112121py** [comment]
聽歌真的蠻幫助恢復心情的，無論是在開心或難過的時候，當然聽的類型也會蠻不一樣，但蠻有趣的是， 如果很專注在某件正在做的事情上面時，就幾乎不會管耳機裡播的歌是甚麼了xd，我的youtube music就會一直隨機撥放，然後幾乎不怎麼會聽，真的變成bgm，但還不錯的點是，這樣偶爾會挖到一些寶藏歌曲

**Comment by forestdew507** [comment]
音樂和性格之間的聯繫真的滿讓我好奇的，有些想認識的人但不好意思去認識，就開始聽對方的歌單XD 雖然不可能因為幾首歌就猜透，但也因為這樣會去思考兩者之間的關聯性是什麼。

**Comment by Tchang323** [comment]
聽音樂讓我的心情變得更愉悅，而我也發現自己喜歡的音樂風格有一定的共通點。我偏愛那種帶有厚度的聲音，能唱出讓人放鬆的氛圍。像是 Billie Eilish 的歌，她的歌聲低沉而有層次，能夠傳遞出獨特的情感，讓人感受到深深的舒壓效果。

**Comment by wcyeee** [comment]
我也是只要有可以聽音樂的機會就一定要塞耳機的人！但我在讀書的時候好像沒有辦法聽有歌詞的歌，會一直想聽歌詞所以讀書讀一讀就會開始很不專心，所以也會常常聽Spotify上的咖啡廳音樂播放清單，聽著聽著也覺得心情有變輕鬆！

**Comment by Karsta0216** [comment]
我自己在不同情況下聽的音樂會不一樣，歌單是依照情境去分類的。認真的時候聽可以專注的音樂、打遊戲要操作的時候聽一些epic的音樂、放空的時候聽一些比較輕靈空泛的音樂等等。

**Comment by Teddy111802518** [comment]
我覺得音樂是一個很不一樣的東西，他沒有實體，但你卻能感受到他，我暑假去維也納聽莫札特的音樂會，真的不得不佩服莫札特這個神童，你如果把眼睛閉起來用心感受，可以感覺到每個音符每首歌不同的情緒，而且因人而異，就算妳的感受跟作曲家想表達的不一樣，也沒有對錯，音樂是沒有對錯的

**Comment by yue083139** [comment]
我平常喜歡邊聽音樂邊做事情，因為聽音樂的時候總是能讓我專心，有趣的是，原本是因為不想要聽自己聽得懂的語言所以都聽日文歌，後來聽一聽就不小心考到n1好像也沒用了，總之聽音樂確實能在我情緒低落的時候給我一點情緒價值，透過去感受每首音樂的意境。
